### ISSUE TITLE
Brain Decoding: Toward Real-time Reconstruction of Visual Perception

### ISSUE THREAD

**Issue by chaerinkong** [chaerinkong/paper-reviews#229]
# TL; DR

- Learns MEG decoder that maps brain signals (MEG) to image latents for image retrieval / generation.

# Motivation

- Image reconstruction from fMRI signal presented promises, but they inherently lack temporal bandwidth (~0.5 Hz), limiting their real-life applications.
- MEG has higher temporal resolution (~5000 Hz) but spatial clarity falls behind.

# Formulation

![image](https://github.com/reyllama/paper-reviews/assets/46308407/68973418-031e-49f9-b56e-867e6ccbbc43)

- Experiment with various image encoders, such as VGG-19, CLIP, DINOv2, etc.
- Combines contrastive loss (retrieval loss) and regression loss (MSE) to learn the **brain module**.

- **SPECIAL NOTES ON DATA PREPROCESSING**
![image](https://github.com/reyllama/paper-reviews/assets/46308407/47907438-56cb-4a69-96ee-1e6d62a41e72)

# Evaluation

![image](https://github.com/reyllama/paper-reviews/assets/46308407/713e446d-fc67-4a38-860f-f38e19aa4124)
![image](https://github.com/reyllama/paper-reviews/assets/46308407/bb514d00-121d-4bed-8b61-7bb50299e20e)
![image](https://github.com/reyllama/paper-reviews/assets/46308407/fda5a38d-8d21-42dc-ace3-65faef6a99e2)
![image](https://github.com/reyllama/paper-reviews/assets/46308407/a901a5cb-9287-4454-9578-52a73028927f)
![image](https://github.com/reyllama/paper-reviews/assets/46308407/a99d82d3-83f7-45d4-a416-a511da897d97)
![image](https://github.com/reyllama/paper-reviews/assets/46308407/6a4486d1-384f-4158-b786-c20b14eae54b)
![image](https://github.com/reyllama/paper-reviews/assets/46308407/c534cdc6-fcab-44b3-a550-7c6f3e24f2fd)

# Inspiration

- Preprocessing of MEG signals.
